libclc/amdgcn-amdhsa/libspirv/group/collectives.cl

```diff
@@ -150,7 +153,38 @@ __CLC_SUBGROUP_COLLECTIVE(FMax, __CLC_MAX, float, f, -FLT_MAX)
 __CLC_SUBGROUP_COLLECTIVE(FMax, __CLC_MAX, double, d, -DBL_MAX)
 
 __CLC_SUBGROUP_COLLECTIVE(All, __CLC_AND, bool, a, true)
-__CLC_SUBGROUP_COLLECTIVE(Any, __CLC_OR, bool, a, true)
```

Comment:
Just keeping the changes in here so the E2E tests pass (they will fail without). If the other one gets merged then this will disappear from diff I think